app/views/category/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for(category) do |f| %>
+  <% if category.errors.any? %>
+    <div>
+      <h2><%= pluralize(category.errors.count, "error") %> prohibited this category from being saved:</h2>
+
+      <ul>
+        <% category.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
```

Comment:
Please remove line 2-8. We have flash messaging built into application.html.erb, messages should be rendered there